### PR TITLE
prepared release of version 2.8.0

### PR DESCRIPTION
**Pull request by dakshvar22**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.mdx [hunk 1]
```diff
@@ -17,6 +17,13 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.0] - 2021-07-08
```
Comment (joejuzl) [inline CHANGELOG.mdx:20]: Hmm seems we are missing a changelog for UnexpecTED?

**Review by joejuzl — CHANGES_REQUESTED** [review]

**Comment by dakshvar22** [comment]
This is rasa-sdk 😅

File: CHANGELOG.mdx (hunk 1 above)
Comment (dakshvar22) [reply CHANGELOG.mdx:20]: This is rasa-sdk 😅

Comment (joejuzl) [reply CHANGELOG.mdx:20]: haha oh yeah 🤦

**Review by joejuzl — APPROVED** [review]

**Pull request merged**